bedrock/legal/templates/legal/terms/firefox-private-network.html

```diff
@@ -95,9 +95,9 @@ <h2>{{ _('Permissions Necessary to Provide the Service') }}</h2>
     <h2>{{ _('Browser Protection') }}</h2>
     <p>
       {% trans %}
-      When you sign up for FPN Browser Protection, you get four passes per month. Each time you turn FPN Browser
-      Protection on, you claim one of your passes and it runs for eight hours without stopping. Once you use all four
-      passes, you will not be able to use FPN Browser Protection until the next month, when you receive 4 new passes.
+      When you sign up for FPN Browser Protection, you get a certain number of per month. Each time you turn FPN Browser
```

Comment:
Missing the word "passes" here, I think?

```suggestion
      When you sign up for FPN Browser Protection, you get a certain number of passes per month. Each time you turn FPN Browser
```